lang/c++/api/NodeConcepts.hh

```diff
@@ -47,33 +47,29 @@ namespace avro {
 
 namespace concepts {
 
-template <typename Attribute>
-struct NoAttribute
-{
+template<typename Attribute>
+struct NoAttribute {
     static const bool hasAttribute = false;
 
     size_t size() const {
         return 0;
     }
 
-    void add( const Attribute &attr) {
+    void add(const Attribute &attr) {
         // There must be an add function for the generic NodeImpl, but the
         // Node APIs ensure that it is never called, the throw here is
         // just in case
         throw Exception("This type does not have attribute");
     }
 
-    const Attribute &get(size_t index = 0) const {
+    const Attribute &get(size_t = 0) const {
```

Comment:
it would probaby be more readable to do something like `get(size_t /*index*/ = 0`

Reply:
Good point. Will fix it.